config/default/contact.form.contact.yml

```diff
@@ -2,6 +2,9 @@ uuid: e6cc3d12-a2bb-425a-87c6-6c3cae72d701
 langcode: en
 status: true
 dependencies: {  }
+third_party_settings:
```

Comment:
@charlieartist Unless you made a change to the contact form, I don't think this config file should have been committed. @aweingarten ?